lib/mutant/repository.rb

```diff
@@ -20,17 +20,9 @@ def call(subject)
 
     # Diff between two objects in repository
     class Diff
-      include Adamantium, Concord.new(:from, :to)
+      include Adamantium, Anima.new(:config, :from, :to)
```

Comment:
One other thing to keep in mind. If `config` is a required argument to this, does it make sense for the `Config` object to have helper methods that can be used to create a diff, passing itself into `Diff.new`? I don't know if it does, but I figured I should ask.